algorithms_task_2/student/include/impl.h

```diff
@@ -9,4 +10,13 @@
  *
  * @return true if source starts from a prefix
  */
-bool startsWith(const std::string& source, const std::string& prefix);
+bool startsWith(const std::string& source, const std::string& prefix) {
+    if (prefix.empty()) {
+        return prefix.empty() || source.empty();
```

Comment:
why do you need to check both? it will be evaluated to true

Reply:
my mistake